tsconfig.json

```diff
@@ -2,6 +2,8 @@
   "extends": "@tsconfig/node18/tsconfig.json",
   "compilerOptions": {
     "module": "commonjs",
+    "esModuleInterop": true,
```

Comment:
Are these still needed? Why do we need them now?

esmoduleinterop and allowjs

```suggestion


```

Reply:
eslint was yelling at me until i added esmoduleinterop. are you using eslint or nah?

Reply:
huh it stopped, not sure why. okay i removed